### ISSUE TITLE
It is recommended to add a new project that can create a clean-architecture restful API structure

### ISSUE THREAD

**Issue by MuhmdHsn313** [kataras/iris-cli#8]
Hello
It is recommended to add a new project that can create a clean-architecture restful API structure.
Thanks

**Comment by kataras** [comment]
Yes, I can create a template for clean arch with Iris + Go. There is already a working example at: https://github.com/iris-contrib/clean-architecture

**Comment by MuhmdHsn313** [comment]
Thanks a lot, I'm using it, but I mean you should add it to iris-cli.
